pkgs/by-name/ma/managarr/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "managarr";
+  version = "0.4.2";
+  src = fetchFromGitHub {
+    owner = "Dark-Alex-17";
+    repo = "managarr";
+    tag = "v${version}";
+    hash = "sha256-OxGFubtMsGnR8cWDKkeAgryY095uydA3LzE5SS0dspQ=";
+  };
+
+  cargoHash = "sha256-bws4LwpJGfq0hCA2Fq51uCMvr0arbayppq1jvDrILZI=";
+
+  nativeBuildInputs = [ perl ];
+
+  meta = {
+    description = "A TUI and CLI to manage your Servarrs";
```

Avoid leading articles (see pkgs/README.md)
```suggestion
    description = "TUI and CLI to manage your Servarrs";
```